docs/modules/concepts/pages/glossary.adoc

```diff
@@ -0,0 +1,31 @@
+= Glossary
+
+xref:mail/messages/imf.adoc[IMF]::
+*Internet Message Format* (IMF) is the base specification that defines the
+syntax of an email message.
+
+IMAP::
+
+JMAP::
+
+xref:mail/messages/mime.adoc[MIME]::
+*Multipurpose Internet Mail Extension* (MIME) is a specification that extends
```

Comment:
we should link to RFCs here in my opinions. what do you think?

Reply:
It does link, but indirectly. The original idea is to link to our description of MIME, which has links to the RFCs. There are many RFCs, so providing a single link is not as easy as it sounds. I didn't want the glossary description to be too long. The intent was to make it more like a jumping point to a more in-depth description, with just a short text here.